components/TokenBalance/TokenDeposit.tsx

```diff
@@ -16,6 +16,8 @@ import { useRealmConfigQuery } from '@hooks/queries/realmConfig'
 import VanillaVotingPower from '@components/GovernancePower/Vanilla/VanillaVotingPower'
 import { DepositTokensButton } from '@components/DepositTokensButton'
 import VanillaWithdrawTokensButton from './VanillaWithdrawTokensButton'
+import { findPluginName } from '@constants/plugins'
+import PythVotingPower, { PYTH_INSTRUCTIONS } from '@components/ProposalVotingPower/PythVotingPower'
 
 /** deposit + withdraw for vanilla govtokens, used only in account view. plugin views still use this for council. */
```

Comment:
I think you should leave this component as vanilla and make a Pyth version instead, and use that. (I didn't see where you actually import this component anyway, but the hierarchy is very confusing here and that's not your fault.)